tools/dependencies-report/src/main/resources/licenseMapping.csv

```diff
@@ -26,7 +53,10 @@ dependency,dependencyUrl,licenseOverride
 "org.codehaus.janino:commons-compiler:3.0.8",z,BSD-3-Clause-Attribution
 "org.codehaus.janino:janino:3.0.8",z,BSD-3-Clause-Attribution
 "com.fasterxml.jackson.dataformat:jackson-dataformat-cbor:2.9.5",z,Apache-2.0
+"com.github.jnr:jnr-unixsocket","https://github.com/jnr/jnr-unixsocket",Apache-2.0
+"com.github.jnr:jnr-x86asm","https://github.com/jnr/jnr-x86asm",MIT
 "org.jruby:jruby-complete:9.1.13.0",z,EPL-2.0|Ruby
+"org.jruby:jruby-complete:9.1.14.0",z,EPL-2.0|Ruby
```

Comment:
will auto validation fail since the notice doesn't include "complete"? (it is named `jruby-9.1.14.0-NOTICE.txt`)

Reply:
url: http://jruby.org/